a_tour_of_textile.yml

```diff
@@ -0,0 +1,472 @@
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    like_add:
+      cmd:
+        code: 'textile likes add --block="QmVoKpKsg5MkW11bK3LVmd3xMMaxTutVime32sV6EZWeLk"'
+      js_http_client:
+        code: 'Textile.likes.add("QmUvWjstQzR6y7UctRJgVjcKsKzutZoiBsQw6WBXMnmg84")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    files_ls:
+      cmd:
+        code: 'textile files ls --thread="12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw"'
+      js_http_client:
+        code: 'Textile.files.list("12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    feed:
+      cmd:
+        code: 'textile feed --thread="12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw"'
+      js_http_client:
+        code: 'Textile.feed.get("12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    add_location_data:
+      cmd:
+        code: "echo '{ \"latitude\": 48.858093, \"longitude\": 2.294694 }' | textile files add --thread=\"12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw\"'"
+      js_http_client:
+        code: 'Textile.files.add(JSON.stringify({ \"latitude\": 48.858093, \"longitude\": 2.294694 }), "", "12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    add_location_data_fail:
+      cmd:
+        code: "echo '{ \"latitude\": 91, \"longitude\": 2.294694 }' | textile files add --thread=\"12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw\"'"
+      js_http_client:
+        code: 'Textile.files.add(JSON.stringify({ \"latitude\": 91, \"longitude\": 2.294694 }), "", "12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    thread_leave:
+      cmd:
+        code: 'textile threads rm "12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw" --api="http://127.0.0.1:41600"'
+      js_http_client:
+        code: 'Textile.threads.remove("12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+core:
+  daemon:
+    cmd:
+      code: 'textile daemon --repo-dir="/tmp/buddy"'
+  logs:
+    cmd:
+      code: 'textile logs'
+  logs_text_only:
+    cmd:
+      code: 'textile logs --tex-only --level="info"'
+  init:
+    cmd:
+      code: 'textile init --seed="$(textile wallet init | tail -n1)" --repo-dir="/tmp/buddy" --swarm-ports="4101" --api-bind-addr="127.0.0.1:41600" --gateway-bind-addr="127.0.0.1:5150"'
+    js_http_client:
+      code: 'TODO'
```

Comment:
missing?

Reply:
The purpose of this PR wasn't to complete the JS entries nor any of the other TODOs obviously missing here (and inline previous to this pr), it was to setup the YAML reading method to make it easy to start adding those.

Reply:
Ah, ok. I though I gave a thumbs up (no review was requested) back when it was just the YAML change. Seemed like you were trying to jam in all the js-http methods here.

Reply:
kk, nah, i just hit all the ones i could pull from current lib simply. some of the missing js ones became tickets others are unclear to me the right method